### PR TITLE
feat(Downloads): created the downloads page and added some more changes

### LINKED ISSUE #6: יצירת עמוד הורדות קבצים חשובים
image

בלחיצה על כפתור ההורדות:
image

נפתח מסך המציג כפתורי הורדה הנראים כך - 
image - system_update_alt
ניתן למצוא אותו כאן: https://material.io/resources/icons/?search=down&icon=system_update_alt&style=baseline

-בכל לחיצה על הכפתור מורד קובץ ששמו רשום בתחתית המסגרת
-בעת לחיצה ובhover על כפתור ההורדה, הכפתור יוקף בעיגול אפרפר (לאחר הלחיצה יחזור האייקון למצבו הטבעי בלי מסגרת- לדוגמא טופס המק פורק)

### PR DESCRIPTION
**Pull request by MayaSasson**
The additional changes:
- Imported the fonts to the new project
- Changed some of the files structure

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
     <title><%= htmlWebpackPlugin.options.title %></title>
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900">
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@mdi/font@latest/css/materialdesignicons.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
```
Comment (RoeiRom) [inline public/index.html:11]: Why is this exist?

File: src/components/CalendarComponents/categoriesBar.vue [hunk 2]
```diff
@@ -71,7 +71,6 @@ export default class CategoriesBar extends Vue {
         display: flex;
         justify-content: space-between;
         align-items: center;
-        width: 90%;
```
Comment (RoeiRom) [inline src/components/CalendarComponents/categoriesBar.vue:74]: Why take it off?

File: src/views/Downloads.vue [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+@Component({
+  apollo: {
+    allDownloads: {
+      query: AllDownloads,
+    },
+  },
+  data() {
+    return {
+      publicPath: process.env.BASE_URL,
```
Comment (RoeiRom) [inline src/views/Downloads.vue]: You can put the data inside the class, the target of putting it in is making the component OOP...

File: public/index.html (hunk 1 above)
Comment (MayaSasson) [reply public/index.html:11]: It is the way to import the icons I added to the project

File: src/components/CalendarComponents/categoriesBar.vue (hunk 2 above)
Comment (MayaSasson) [reply src/components/CalendarComponents/categoriesBar.vue:74]: Because it causes the categories bar to be smaller than the calendar

**Review by RoeiRom — APPROVED** [review]

**Pull request merged**